### DISCUSSION TITLE
[resolved] Severe problem after upgrading to 6.0.10 – Site won’t load anymore

### DISCUSSION THREAD

**Discussion by retemirabile** [snipe/snipe-it#11817] [General]
Hi, 
I upgraded to 6.0.10 yesterday and after the Upgrade, my site doesn’t load anymore – at all. I don‘t get an error message, just an empty page: 

https://assets.kgn.schule/

* Upgraded via Git (as I’ve done several times before without problems). 
* PHP 7.4.28 (can’t go to 8 yet because of some other software)
* GNU/Linux, “latest stable version” according to my webhoster (I’m on shared hosting)

The upgraded ended with this message: 

```
---------------------- FINISHED! -----------------------
All done! Clear your browser cookies and re-login to use 
your upgraded Snipe-IT!
--------------------------------------------------------

```

Here’s the full log of me running the update
[snipeit-update-log_2022-09-14.txt](https://github.com/snipe/snipe-it/files/9563333/snipeit-update-log_2022-09-14.txt)

I’d greatly appreciate any help! 
Thanks. 
Andreas

**Comment by mbachani-art** [comment]
Did you try lookin in /var/log/apache2/error logs and see what it says?
Restarted apache server?

**Comment by retemirabile** [comment]
Thanks for your tip. I’m on shared hosting so I don’t have access to those logs. But I did restart apache – didn’t help.

**Comment by retemirabile** [comment]
@snipe said on discord to install composer. I managed to do that in my snipe directory using [these instructions](https://getcomposer.org/download/), then ran php update.php again and now my Snipe install works again.
